test_api/07_test_schemas.py

```diff
@@ -0,0 +1,23 @@
+import json
+import pytest
+from jsonschema import validate
+from test_api.constant import SchemasConsts
+
+
+def assert_valid_schema(data, schema_file):
+    with open(schema_file) as f:
+        schema = json.load(f)
+    return validate(instance=data, schema=schema)
+
+
+def test_get_todos(create_session, workplace):
+    res = create_session.get(url=f'{workplace}/1')
+    assert_valid_schema(res.json(), SchemasConsts.PATH_TO_SCHEMA)
+
+
+@pytest.mark.parametrize('schema', [
+    SchemasConsts.PATH_TO_SCHEMA_FILE
+])
+def test_get_todoses(create_session, workplace, schema):
+    res = create_session.get(url=workplace)
+    assert_valid_schema(res.json(), schema)
```

Comment:
You can pass **SchemasConsts.PATH_TO_SCHEMA_FILE** directly to the test without using @pytest.mark.parametrize

```
def test_get_todoses(create_session, workplace):
    res = create_session.get(url=workplace)
    assert_valid_schema(res.json(), SchemasConsts.PATH_TO_SCHEMA_FILE)
```

Reply:
added